README.md

```diff
@@ -1,13 +1,13 @@
-# Asha Pipewire Sink
-A sample asha audio implementation designed to work with pipewire and bluez.
+# ASHA (Audio Streaming for Hearing Aids) Pipewire Sink
```

Comment:
I thought ASHA was American Speech-Language-Hearing Association... The google doc agrees with you though?

Reply:
That's why I was careful to refer to "ASHA audio". If I had it wrong, then "ASHA Audio" is redundant.

Reply:
That google doc is... problematic. It doesn't entirely match the implementation and has stuff in it that I haven't actually seen implemented in practice. I wouldn't fret over a singular hearing aid :D